src/libraries/System.Private.CoreLib/src/System/Collections/HashHelpers.cs

```diff
@@ -109,5 +109,19 @@ public static uint FastMod(uint value, uint divisor, ulong multiplier)
             Debug.Assert(highbits == value % divisor);
             return highbits;
         }
+
+        // returns 2^x >= size
+        public static int AlignToPowerOfTwo(int size)
```

Comment:
Should use https://github.com/dotnet/runtime/blob/21734a4134dddd977b45e6a39cb139e6b79ea7e9/src/libraries/System.Private.CoreLib/src/System/Numerics/BitOperations.cs#L76 ?
(https://github.com/dotnet/runtime/issues/43135 for reference).

Reply:
@gfoidl This was written long time ago. I suspected we have LZCNT-based helper for this by now when porting over, but did not have time to look it up yet. Thanks!!